_maps/map_files/Deltastation/DeltaStation2.dmm

```diff
@@ -102671,6 +102671,34 @@
 	},
 /turf/open/floor/circuit/green,
 /area/engine/supermatter)
+"YGM" = (
```

Comment:
Not required, but if you wouldn't mind could you condense these down to one key?